apps/api-documenter/src/cli/BaseAction.ts

```diff
@@ -15,12 +15,15 @@ import {
   IResolveDeclarationReferenceResult
 } from '@microsoft/api-extractor-model';
 
-export abstract class BaseAction extends CommandLineAction {
-  protected inputFolder: string;
-  protected outputFolder: string;
+export interface IBuildApiModelResult {
```

Comment:
The strict compiler check is making our code worse here.  (If there isn't an easy way to eliminate `IBuildApiModelResult`, I'm okay with keeping the worse code here, since `BaseAction` isn't a particularly important class.)